### ISSUE TITLE
Ensure compatiblity between carl of pycarl and carl of Storm

### ISSUE THREAD

**Issue by volkm** [moves-rwth/stormpy#82]
- We should check both carl versions (used in pycarl and Storm) to ensure they are the same. (Maybe even check the `carl_DIR` used during compilation?)
- It is possible that pycarl has disabled the cln support and Storm is compiled with CLN. This currently yields an ImportError. It would be good to catch it already at the build stage.

**Comment by sjunges** [comment]
Isnt this fixed?

**Comment by volkm** [comment]
No, not yet. We are currently checking that the pybind version is the same between pycarl and stormpy (cf. https://github.com/moves-rwth/stormpy/pull/86). We are however not yet checking that the carl version (and the support for CLN) between pycarl and Storm is the same.

**Comment by volkm** [comment]
Fixed after integrating pycarl into stormpy in https://github.com/moves-rwth/stormpy/pull/205
